test/lib/code_corps/messages/conversation_parts_test.exs

```diff
@@ -13,9 +13,44 @@ defmodule CodeCorps.Messages.ConversationPartsTest do
 
   describe "create_changeset/2" do
     test "with valid attributes" do
+      insert(:user, id: 1)
+      insert(:conversation, id: 1)
       attrs = @valid_attrs |> Map.merge(%{author_id: 1, conversation_id: 1})
+      {:ok, conversation} = ConversationParts.create_changeset(%ConversationPart{}, attrs) |> Repo.insert
+      assert conversation.body == "Test body."
+      assert conversation.part_type == "comment"
```

Comment:
I think you also probably want to assert the conversation.user_id here is equal to the user.id.

Reply:
done